docs/root/operations/runtime.rst

```diff
@@ -16,3 +16,21 @@ Due to this usage of runtime guards, some deployments might find it useful to se
 dynamic runtime configuration as a safety measure to be able to quickly disable the new behavior
 without having to revert to an older version of Envoy or redeploy it with a new set of static
 runtime flags.
+
+Runtime Keys
+------------
+
+Currently the below runtime keys are provided.
```

Comment:
There are lots more runtime keys than this. Can we actually document this key with the heap shrink extension where it's implemented? Thank you.

/wait

Reply:
Sorry, what is the heap shrink extension? Is it [the action in overload manager](https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/overload_manager/overload_manager#overload-actions)?

This runtime key doesn't control the behavior of that overload action. 
Instead, this runtime key is controlling how aggressive an active shrink should be.
The *active shrink* occurs at "receiving xds response" and "receiving any admin request".

Reply:
I see, OK. Can we document it somewhere near the xDS code or maybe here? https://www.envoyproxy.io/docs/envoy/latest/operations/performance This feels like the wrong place.

Reply:
Performance looks like the best place. I will revert the runtime key summary and add a shrink section in performance.